projectgenerator/gui/generate_project.py

```diff
@@ -181,3 +192,18 @@ def link_activated(self, link):
 
         else:
             QDesktopServices.openUrl(link)
+
+    def select_crs(self):
+        """ Open a dialog to select CRS """
+        crsSelector = QgsProjectionSelectionDialog(self)
+        crsSelector.setCrs(QgsCoordinateReferenceSystem(self.epsg, type=QgsCoordinateReferenceSystem.PostgisCrsId))
+        res = crsSelector.exec()
```

Comment:
Hmm, I thought that's called `exec_()`for some name collision or reserved keyword, but you probably tested it...